pipelines/build/openjdk11_pipeline.groovy

```diff
@@ -166,6 +166,13 @@ def buildConfigurations = [
                 additionalFileNameTag: "linuxXL",
                 configureArgs        : '--with-noncompressedrefs --disable-ccache'
         ],
+        riscv64Linux      :  [
+                os                   : 'linux',
+                additionalNodeLabels : 'riscvcross'
+                arch                 : 'riscv'
```

Comment:
Comma at the end of this line and the line above ;)

Reply:
👍